src/main/webapp/index.html

```diff
@@ -19,26 +19,72 @@
   <head>
     <meta charset="UTF-8">
     <title>CodeU 2019 Starter Project</title>
-    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="css\main.css">
```

Comment:
for the stylesheet, i think one would be enough. If the two files have overlap ids, it may cause a problem.

Reply:
I think we need to have two style sheet because other pages use main.css as well, so I cannot add styling that is specific to index.html to main because that is used by other pages as well